src/js/Router.js

```diff
@@ -18,12 +18,8 @@ export default class Router extends String {
         this._config = { ...this._config, absolute };
 
         if (name) {
-            if (!this._config.routes[name]) {
-                throw new Error(`Ziggy error: route '${name}' is not in the route list.`);
-            }
-
-            this._route = new Route(name, this._config.routes[name], this._config);
-            this._params = this._parse(params);
+            this._name = name;
```

Comment:
I'm trying to get the public methods to not depend on `this` so only the passed params are stored and this error now happens in resolve()